### PR TITLE
Church address map

### PR DESCRIPTION
**Pull request by Opio-Emmanuel-Omona**

### CODE REVIEW

File: resources/views/includes/search_bar.blade.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="col-md-6 mx-auto">
+  <form>
+    <input class="col-md-10 mb-2" type="text" placeholder="Search.." name="search">
```
Comment (naush) [inline resources/views/includes/search_bar.blade.php]: Nitpick: three dots instead of two 😉

File: tests/Feature/ChurchPresenterTest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  protected $churches;
+  protected $church_1;
+  protected $church_2;
+
+  public function setUp(): void
+  {
+    parent::setUp();
+    $this->church_1 = factory(Church::class)->create();
+    $this->church_2 = factory(Church::class)->create();
+    $this->churches = collect([$this->church_1, $this->church_2]);
```
Comment (naush) [inline tests/Feature/ChurchPresenterTest.php:24]: I'm curious what does `collect` do?

File: tests/Feature/PermissionControllerTest.php [hunk 3]
```diff
@@ -38,6 +38,49 @@ public function test_permission_index_has_users()
...
+  {
+    $this->withoutExceptionHandling();
+    $response = $this->get(route('permissions.index', ['email' => $this->user->email]));
+    $response->assertViewHas('users');
+    $this->assertEquals($this->user->id, $response['users']->first()->id);
+  }
+
+  public function test_permission_search_email_ignores_case()
+  {
+    $response = $this->get(route('permissions.index', ['email' => strtolower($this->user->email)]));
```
Comment (naush) [inline tests/Feature/PermissionControllerTest.php]: Does the email have uppercase letters?

**Comment by naush** [comment]
A few nitpicks but overall LGTM 👍

File: tests/Feature/ChurchPresenterTest.php (hunk 2 above)
Comment (Opio-Emmanuel-Omona) [reply tests/Feature/ChurchPresenterTest.php:24]: collect defines a collection, which is what a where query returns

File: tests/Feature/PermissionControllerTest.php (hunk 3 above)
Comment (Opio-Emmanuel-Omona) [reply tests/Feature/PermissionControllerTest.php]: Actually Good catch, let me fix that

**Pull request merged**